src/cmds/qsub.c

```diff
@@ -162,207 +162,179 @@
 
 #define MAX_QSUB_PREFIX_LEN 32
 #define QSUB_DMN_TIMEOUT_LONG 60  /* timeout for qsub background process */
-#define QSUB_DMN_TIMEOUT_SHORT 5  
+#define QSUB_DMN_TIMEOUT_SHORT 5
 
-static char PBS_DPREFIX_DEFAULT[] = "#PBS";
 
-/* globals */
+/* extern global variables */
 #ifdef NAS /* localmod 005 */
 extern void set_attr_resc(struct attrl **attrib, char *attrib_name, char *attrib_resc, char *attrib_value);
 #endif /* localmod 005 */
 extern char *msg_force_qsub_update;
-int comm_sock;
 
-/* Socket for x11 communication */
-int X11_comm_sock;
+/*
+ * The following variables are file static (ie. global in this file).
+ */
+static char PBS_DPREFIX_DEFAULT[] = "#PBS";
```

Comment:
I changed PBS_DPREFIX_DEFAULT to be a #define because it is used as a return value in a function, which prevents it from being defined as a const. However, I kept the other strings as static const in order to limit their scope to this file. Does this inconsistency between #define and const usage seem reasonable?

Reply:
If you put the #define directly in qsub.c, it will limit the scope to just qsub.c.  If you do change them to #defines make them uppercase.